contrib/kube-prometheus/jsonnet/kube-prometheus/alerts/prometheus-operator.libsonnet

```diff
@@ -0,0 +1,50 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'prometheus-operator',
+        rules: [
+          {
+            alert: 'PrometheusOperatorAlertmanagerReconcileErrors',
+            expr: |||
+              rate(prometheus_operator_alertmanager_reconcile_errors_total{%(prometheusOperatorSelector)s}[5m]) > 0.1
+            ||| % $._config,
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              message: 'Errors while reconciling Alertmanager in {{$labels.namespace}} namespace.',
```

Comment:
Nit: in the kubernetes-monitoring mixins the template variables all have spaces between the brackets and the variable name (look at the generated alerting rules below. Functionally of course it it the same but it would be nice to standardize

Reply:
Also, the kubernetes monitoring mixins all capitalize the kubernetes object types, so “namespace” becomes “Namespace”, “daemonset” becomes “DaemonSet” etc